src/meta/test/ProcessorTest.cpp

```diff
@@ -1155,115 +1464,6 @@ TEST(ProcessorTest, AlterEdgeTest) {
     }
 }
 
-
-TEST(ProcessorTest, SameNameTagsTest) {
```

Comment:
Why remove the UT?

Reply:
not remove
Just integrated it into line 296, line 328.
This makes the test case clearer.